src/test/java/com/deloitte/ddwatch/services/ProjectServiceTest.java

```diff
@@ -160,9 +159,6 @@ public void findByTagShouldReturnATag(){
 
         //assert
         assertNotNull(projectsReturned);
```

Comment:
This is the right assertion 😀
`assertEquals(tag.getProjects(), projectsReturned);`